web3.js/rollup.config.types.js

```diff
@@ -4,4 +4,5 @@ export default {
   input: './declarations/index.d.ts',
```

Comment:
The bug is that you moved `./declarations/index.d.ts` to `./declarations/src/index.d.ts` with the inclusion of the `test/` directory. You either need to make that not happen, or update `rollup.config.types.js`.

Reply:
Delete your `declarations/` directory and run the script again and you'll see the problem.